caliper_sender.py

```diff
@@ -0,0 +1,127 @@
+import caliper
+import psycopg2
+# from dotenv import load_dotenv
+import requests, json, sys, os, logging
+from datetime import datetime, date, time
+
+import os
+
+# load_dotenv()
+
+def send_caliper_event():
+    print("Connect to database...")
+    conn = psycopg2.connect(
+        dbname = "runestone",
+        user = os.getenv("USER"),
+        password = os.getenv("PASSWORD")
+        )
+    cur = conn.cursor()
+
+    # create a runtime table, get the last runtime
+    try:
+        cur.execute("CREATE TABLE IF NOT EXISTS execute_time (runtime timestamp NOT NULL, status varchar(10) NOT NULL);")
+        now = datetime.utcnow()
+        event_time = now.strftime('%Y-%m-%d %H:%M:%S')
+        cur.execute("INSERT INTO execute_time (runtime, status) VALUES ('{}', 'completed');".format(event_time))
```

Comment:
The insert call does not happen at the end of "send Caliper event" process, it is at the beginning.

I would argue for the opposite.

Reply:
@zqian Just want to confirm if I understood this right. You mean the insertion should happen at the end of the caliper sending event, right?